plugins/parsers/csv/parser.go

```diff
@@ -69,6 +136,10 @@ func NewParser(c *Config) (*Parser, error) {
 		return nil, fmt.Errorf("csv_column_names field count doesn't match with csv_column_types")
 	}
 
+	if err := c.initializeMetadataSeparators(); err != nil {
+		return nil, fmt.Errorf("Failed initializing separators. %v", err)
```

Comment:
Error messages should start with a lowercase letter. How about
```suggestion
		return nil, fmt.Errorf("initializing separators failed: %v", err)
```